### DISCUSSION TITLE
Is there any reason NOT to enable ElevationShim?

### DISCUSSION THREAD

**Discussion by emendelson** [otya128/winevdm#1323] [Q&A]
I've written an Inno Setup installer for otvdm, and, for the next release of the installer, I want to include isfixload.exe and isfix32.dll. The installer can set the compatibility mode for isfixload.exe and can write the full path into otvdm.ini. My question is: should it enable EmulationShim= or leave the line commented out? 

It seems to be easier to enable the line than to write in the full path and then comment it out, so I hope there's no strong reason for NOT enabling it....

**Comment by cracyc** [comment]
I think that shouldn't be a problem.

**Reply by emendelson** [reply]
Thank you!
